source/actions.js

```diff
@@ -1,14 +1,17 @@
 
 import {
+    START,
     SET,
     SET_PROFILE,
     LOGIN,
     LOGOUT,
     LOGIN_ERROR,
-    NO_VALUE
+    NO_VALUE,
+    INIT_BY_PATH
 } from './constants'
 
 import { Promise } from 'es6-promise'
+var moment = require('moment');
```

Comment:
moment is better because it have built in function for diff, for example isSame(), the idea behind this indicators is to check loading or use in shouldComponentUpdate